src/main/java/com/amazon/ion/impl/_Private_IonTextWriterBuilder_1_0.java

```diff
@@ -0,0 +1,125 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.ion.impl;
+
+import com.amazon.ion.IonCatalog;
+import com.amazon.ion.IonSystem;
+import com.amazon.ion.IonWriter;
+import com.amazon.ion.SymbolTable;
+import com.amazon.ion.system.IonSystemBuilder;
+import com.amazon.ion.util._Private_FastAppendable;
+
+import java.io.OutputStream;
+
+import static com.amazon.ion.impl._Private_Utils.initialSymtab;
+
+/**
+ * Contains configuration for Ion 1.0 text writers.
+ * NOT FOR APPLICATION USE!
+ */
+public class _Private_IonTextWriterBuilder_1_0 extends _Private_IonTextWriterBuilder {
```

Comment:
The contents of this class were movied from `_Private_IonTextWriterBuilder` unchanged.